portfolio/src/main/webapp/index.html

```diff
@@ -28,5 +65,24 @@ <h3>Hello! My name is Andrew Pham and I am a second year at the University of Ca
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      -->
   </body>
+
+ <script>
+        var coll = document.getElementsByClassName("collapse");
```

Comment:
I'd ask Scott or Matt about whether or not you should use var for your js file. I know they were saying that var can be troublesome depending on the complexity of your file and that "let" or "const" could possibly be better options. (but I'm no expert)